### ISSUE TITLE
Rtknavi Qt build error

### ISSUE THREAD

**Issue by anhdung28888** [tomojitakasu/RTKLIB#337]
Hi, I am rebuilding Rtknavi Qt app by VS 2013 using your latest revision (RTKLIB 2.4.3 b29) but no luck.
It said that:
"Error	3	error C2660: 'rtksvrstart' : function does not take 15 arguments	\RTKLIB-rtklib_2.4.3_b29\app\rtknavi_qt\navimain.cpp	1095	1	rtknavi_qt".
Any hint for rebuilding Qt app?

**Comment by JensReimann** [comment]
Please use this tree https://github.com/JensReimann/RTKLIB .
The Qt port in Tomoji's branch hasn't been updated to the latest changes.

**Comment by DavidKelleySCSC** [comment]
JensReimann
    
    Using that tree now. Still got some lib issues to resolve before it
    will run.   A few questions quickly came up...
    
    I presume that RTKLIB/pro is the master project file to build from,
    yes?  Do these need to be shadow builds?
    
    Is there a list of Qt kits this is known to complies with? Due to
    the memory leak that Qt has in a prior versions, we are stuck on
    5.9.1 now, and have to use MSVS rather then GCC in order too use the
    QtWebEngine, so that limits our choices here.  So my big question
    is; Is the branch below known to work with  Qt 5.9.1 with MSVC2017
    64 and/or 32 bit kits.   We could also use MinGW 32bit (5.3.0) if we
    need to but the RTKLIB material I want to try and back port all
    works on MSVC2017 now.  
    
    Please advice, would love to more some of the GUI elements over to
    Qt 
    
    On 9/10/2017 4:12 AM, JensReimann
      wrote:
    
    
      Please use this tree https://github.com/JensReimann/RTKLIB
        .
        The Qt port in Tomoji's branch hasn't been updated to the latest
        changes.
      —
        You are receiving this because you are subscribed to this
        thread.
        Reply to this email directly, view it on GitHub, or mute the thread.
      
        
          
          
        
        
      
      {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/tomojitakasu/RTKLIB","title":"tomojitakasu/RTKLIB","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/tomojitakasu/RTKLIB"}},"updates":{"snippets":[{"icon":"PERSON","message":"@JensReimann in #337: Please use this tree https://github.com/JensReimann/RTKLIB .\r\nThe Qt port in Tomoji's branch hasn't been updated to the latest changes."}],"action":{"name":"View Issue","url":"https://github.com/tomojitakasu/RTKLIB/issues/337#issuecomment-328335855"}}}

**Comment by JensReimann** [comment]
Yes, the RTKLib.pro is your file if you want to compile the whole project.

The codes compiles and works for me using Qt 5.9.0 with MSVC2015/64bit and MinGW 32bit under Windows. I also tested it with gcc and llvm/clang under linux (BTW: the main reason for the port). So, there shouldn't be mature bugs.
Of cause I can't test everything and RTKLib comes without any unit tests (e.g. for checking for memory leaks). Hence, you never can be sure that there is no bug. But if you find anything, I'm glad to fix it.

P.S. All major apps are already ported to Qt (only rtkvplayer & rtkvideo are not working, yet).
